package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "format": "prettier --write \"src/**/*.{js,jsx}\"",
     "lint": "eslint \"src/**/*.{js,jsx}\" --quiet",
-    "dev": "parcel src/index.html"
+    "dev": "parcel src/index.html",
+    "docs": "jsdoc -c jsdoc.conf.json"
```

Comment:
allows us to generate documentation using the CLI. Documentation is opened up in the browser by navigating to `docs/index.html`